wdl/ONTMethylation.wdl

```diff
@@ -242,19 +242,19 @@ task Megalodon {
         samtools cat -o megalodon_results/mappings.bam tmp/*/mappings.bam
         samtools addreplacerg -O BAM -r '@RG\tID:mappings\tSM:~{SM}' megalodon_results/mappings.bam | samtools reheader mappings.header.sam - > megalodon_results/mappings.rh.bam
         samtools sort megalodon_results/mappings.rh.bam > megalodon_results/mappings.sorted.bam
-        samtools index megalodon_results/mappings.sorted.bam
+        samtools index -@${num_cores} megalodon_results/mappings.sorted.bam
 
         ((samtools dict ~{ref_fasta}) && (echo -e '@RG\tID:mod_mappings\tSM:~{SM}')) > mod_mappings.header.sam
         samtools cat -o megalodon_results/mod_mappings.bam tmp/*/mod_mappings.bam
         samtools addreplacerg -O BAM -r '@RG\tID:mod_mappings\tSM:~{SM}' megalodon_results/mod_mappings.bam | samtools reheader mod_mappings.header.sam - > megalodon_results/mod_mappings.rh.bam
         samtools sort megalodon_results/mod_mappings.rh.bam > megalodon_results/mod_mappings.sorted.bam
-        samtools index megalodon_results/mod_mappings.sorted.bam
+        samtools index -@${num_cores} megalodon_results/mod_mappings.sorted.bam
```

Comment:
Is this the right PR for these changes to samtools index to be made?

Reply:
I second this.
And out of curiosity, how much of a performance boost do we gain from using more cores when indexing?